content/en/docs/setup/install/helm/index.md

```diff
@@ -35,12 +50,73 @@ Follow this guide to install and configure an Istio mesh using
     $ helm install istio-base istio/base -n istio-system
     {{< /text >}}
 
+1. Validate the CRD installation with the `helm ls` command:
+
+    {{< text syntax=bash >}}
+    $ helm ls -n istio-system
+      NAME          NAMESPACE       REVISION    UPDATED   STATUS      CHART           APP VERSION
+      istio-base    istio-system    1           ... ...   deployed    base-1.16.1     1.16.1
```

Comment:
Command output here, and in the other following bash commands, should appear as they would on the command line, i.e., without extra indentation.

```suggestion
    NAME          NAMESPACE       REVISION    UPDATED   STATUS      CHART           APP VERSION
    istio-base    istio-system    1           ... ...   deployed    base-1.16.1     1.16.1
```

Reply:
Also, why is the UPDATED value `... ...`, instead of just `...`. Are there really two space-separated values in the output under the UPDATED column?

Reply:
yes, the updated column has space separated values